console/console-init/ui/src/modules/iot-device-detail/components/CredentialsView/ExtensionsView/ExtensionsView.tsx

```diff
@@ -82,7 +82,7 @@ export const ExtensionsView: React.FC<IExtensionsViewProps> = ({
             </Title>
           </GridItem>
           <Table aria-label="extensions view table" cells={columns} rows={rows}>
-            <TableHeader />
+            <TableHeader id="extensions-view-table" />
```

Comment:
Following pattern you have chosen it should be 'extensions-view-table-header / extensions-view-tableheader'

Reply:
Good to go with `id="extensions-view-tableheader"`